src/DripsHub.sol

```diff
@@ -242,6 +192,47 @@ abstract contract DripsHub {
         cycleSecs = _cycleSecs;
     }
 
+    modifier onlyAccountOwner(uint256 userId) {
+        uint32 accountId = uint32(userId >> 224);
+        if (accountId == 0) {
+            // Account ID 0 is for msg.sender verification sub-accounts
+            require(
+                address(uint160(userId)) == msg.sender,
+                "Callable only by the address equal to the user sub-account"
+            );
+        } else {
+            require(
+                accountOwner(accountId) == msg.sender,
+                "Callable only by the owner of the user account"
+            );
+        }
+        _;
+    }
+
+    /// @notice Creates an account.
+    /// Assigns it an ID and lets its owner perform actions on behalf of all its sub-accounts.
+    /// Multiple accounts can be registered for a single address, it will own all of them.
+    /// @return accountId The new account ID.
+    function createAccount(address owner) public virtual returns (uint32 accountId) {
+        DripsHubStorage storage dripsHubStorage = _dripsHubStorage();
```

Comment:
I think we could shorten the naming a bit with the storage variables by changing the structs and helper method.

I would just call the dripsHubStorage `s` everywhere and maybe DripsHubStorage to SlotStorage.
The notation with `s` will be easily understood because you can see it from normally a line above.

The brain can understand such code quicker compared with long names
```solidity
 SlotStorage storage s = _storage();
 accountId = s.lastAccountId + 1;
 s.lastAccountId = accountId;
 s.accountsOwners[accountId] = owner;
```

Reply:
That's a good idea. It seems like we're going to divide the code into libs, so we're going to redo storage access anyway so we pass around the storage reference. Maybe we should apply this improvement then?